projects/ngx-google-analytics/src/lib/directives/ga-event.directive.spec.ts

```diff
@@ -4,7 +4,7 @@ import { NgxGoogleAnalyticsModule } from '../ngx-google-analytics.module';
 import { GaEventCategoryDirective } from './ga-event-category.directive';
 import { GoogleAnalyticsService } from '../services/google-analytics.service';
 import { Component } from '@angular/core';
-import { GaAction } from '../types/ga-action.type';
+import { ActionEnum } from '../enums/google-analytics/action.enum';
```

Comment:
We'll not so major but i think it's better if you rename it `GaActionEnum` to follow the naming convention. 👍

Reply:
Yeah no problem, will check later today